MatzipBook/MatzipBook/Presentation/Home/ViewModel/NearbyRankingCellViewModel.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  NearbyRankingCellViewModel.swift
+//  MatzipBook
+//
+//  Created by 심범수 on 6/29/25.
+//
+
+import UIKit
+
+final class NearbyRankingCellViewModel {
+    
+    private let nearbyRanking: NearbyRanking
+    
+    init(nearbyRanking: NearbyRanking) {
+        self.nearbyRanking = nearbyRanking
+    }
+    
+    var ranking: Int { nearbyRanking.ranking }
+    var restaurantName: String { nearbyRanking.name }
+    var restaurantImage: UIImage? { nearbyRanking.image }
+    var rating: Double { nearbyRanking.rating }
+    var starCount: Int { nearbyRanking.starCount }
+    var formattedDistance: String {
```

Comment:
어짜피 let이라면 
var 로 아래 인스턴스를 뽑아내지 않고 아래와 같이 안하신 이유가 있을까요?!

```suggestion
    
   let nearbyRanking: NearbyRanking
    
    init(nearbyRanking: NearbyRanking) {
        self.nearbyRanking = nearbyRanking
    }
    
    var formattedDistance: String {
```